tests/cases/fourslash/findAllRefsPrivateProperties.ts

```diff
@@ -0,0 +1,24 @@
+﻿/// <reference path='fourslash.ts'/>

```

Comment:
`privateProperties` to me seems ambiguous, given that we have `private` modifier and ES private fields (`#`).

Other ES private field tests have `privateName` in the name. I propose using `privateName` here for consistency, and perhaps a future PR to change the tests to `esPrivate` or `hashPrivate` or `privateIdentifier` or whatever works best.

Reply:
Was not aware of the convention. Renamed the test file.